### PR TITLE
Pettiness - humble suggestions for rephrasing in text documentation

### PR DESCRIPTION
**Pull request by dafnapension**
My humble suggestion for rephrasings in documentation texts

### CODE REVIEW

File: docs/concepts.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ Every operator or pipeline of operators should be saved as an artifact.
 
 Recipe
 -------
-A data prepration recipe is consisted of a sequence of operators.
-The recipe can be easily understood by looking at the list of operations its consisted of.
-The recipe is saved as an artifact and can be shared between different projects
-and allow reproducible and transparent data preparation.
+A data prepration recipe consists of and is defined by a sequence of operators.
```
Comment (yoavkatz) [inline docs/concepts.rst:55]:
```suggestion
A data preparation recipe consists of and is defined by a sequence of operators.
```

**Review by yoavkatz — APPROVED** [review]
Thanks. Looks good.  Please only remove the comments.  BTW, you want to add comments, best to add them in the github conversation.

File: README.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ Unitxt builds on separation. Separation allows adding a dataset, without knowing
 ### 🦄 Simplicity
 Everything is unitxt is simple and designed to feel natural and self-explanatory.
 ### 🦄 Adaptability
-Adding new datasets, loading recipes, instructions and formattors is possible and encouraged!
+Adding new datasets, loading recipes, instructions and formatters is possible and encouraged!
 ### 🦄 Transperancy
-The reosurces and formators of Unitxt are stored as shared datasets and therefore can easily reviewed by the crowd. Moreover, when assembling a dataset with Unitxt, it is very clear to others what's in it.
+The reosurces and formatters of Unitxt are stored as shared datasets and therefore can easily reviewed by the crowd. Moreover, when assembling a dataset with Unitxt, it is very clear to others what's in it.
```
Comment (dafnapension) [inline README.md:39]: I read that formator is more of molding youngster souls. Formatter, as with json formatter, seems more appropriate here.

Comment (dafnapension) [inline README.md:39]: are the formatters, too, stored as datasets?

Comment (yoavkatz) [reply README.md:39]: All formatters, cards,processors, templates, are stored in the catalog:
 
 ls -l src/unitxt/catalog
total 0
drwxr-xr-x   5 yoavkatz  staff   160 Nov  2 10:25 augmentors
drwxr-xr-x   3 yoavkatz  staff    96 Nov  2 10:25 benchmarks
drwxr-xr-x  38 yoavkatz  staff  1216 Nov 12 18:06 cards
drwxr-xr-x   7 yoavkatz  staff   224 Nov 12 15:16 formats
drwxr-xr-x   4 yoavkatz  staff   128 Nov 12 15:16 instructions
drwxr-xr-x  22 yoavkatz  staff   704 Nov 12 15:16 metrics
drwxr-xr-x   3 yoavkatz  staff    96 Nov  2 10:25 operators
drwxr-xr-x  16 yoavkatz  staff   512 Nov 12 15:16 processors
drwxr-xr-x   6 yoavkatz  staff   192 Nov  2 10:25 recipes
drwxr-xr-x   9 yoavkatz  staff   288 Nov  2 10:25 splitters
drwxr-xr-x   5 yoavkatz  staff   160 Nov 12 18:06 tasks
drwxr-xr-x   9 yoavkatz  staff   288 Nov 12 18:06 templates


Can you also fix the type 'reosurces'?

Comment (yoavkatz) [reply README.md:39]: Finally, there is the DCO problem again.  Can you fix it?

Comment (dafnapension) [reply README.md:39]: Thanks, @yoavkatz ,
yes.. will generate a new twin-branch, and copy over..  
I tried to update the commit messages via intellij, but it did not work.

**Comment by yoavkatz** [comment]
Sorry.  DCO again.

**Comment by dafnapension** [comment]
Closing this one for the DCO issue, opened https://github.com/IBM/unitxt/pull/309 instead, implementing all the comments in this one.

**Pull request closed without merging**